lib/src/widgets/typing/ds_typing.widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// Use [align] feft or right to position typing on screen
+class DsTypingWidget extends StatelessWidget {
```

Comment:
to follow the pattern, my name suggestion would be 'ds_typing_message_bubble.widget'

Reply:
Done